openapi/paths/lambdas/lambdas@contentv2@scenes.yaml

```diff
@@ -0,0 +1,50 @@
+get:
+  operationId: getScenes
+  summary: Get Scenes
+  tags:
+    - Lambdas
+  description: Retrieve the list of scenes in a given area
+  parameters:  
```

Comment:
All of this parameters are required

Reply:
Also, the max size of the rectangle is 100x100

Reply:
added